site/logsdb-engine/js/data-gen.js

```
@@ -0,0 +1,511 @@
+    label: "Massive (2M logs, ~34 MB)",
+    count: 2_000_000,
+    durationMinutes: 480,
+    description: "Production scale. Tests memory efficiency.",
+  },
+};
+
+// ── Random Helpers ───────────────────────────────────────────────────
+
+let _seed = 42;
+function mulberry32() {
+  let t = (_seed += 0x6d2b79f5);
```

Biome (2.4.13)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)